--- ../history/1a-baseline-autopipe/mmult_float.cpp	2022-03-06 10:42:30.857284319 +0800
+++ ../history/1b1-pipeline-L3/mmult_float.cpp	2022-03-06 10:42:30.857284319 +0800
@@ -78,6 +78,7 @@
             T tmp = offset_buf[j];
         L3:
             for (int k = 0; k < FEAT; k++) {
+#pragma HLS PIPELINE II = 1
                 tmp += in_buf[i][k] * weight_buf[j][k];
             }
             out_buf[i][j] = tmp;
