--- ../history/1a-baseline-autopipe/mmult_float.cpp	2022-03-06 10:42:30.857284319 +0800
+++ ../history/1b3-pipeline-L1-T2P/mmult_float.cpp	2022-03-06 10:42:30.861284394 +0800
@@ -32,6 +32,11 @@
     T in_buf[BATCH][FEAT];
     T out_buf[BATCH][CLASSES];
 
+#pragma HLS bind_storage variable = offset_buf type = RAM_T2P
+#pragma HLS bind_storage variable = weight_buf type = RAM_T2P
+#pragma HLS bind_storage variable = in_buf type = RAM_T2P
+#pragma HLS bind_storage variable = out_buf type = RAM_T2P
+
     // Input and output AXI stream indices
     int is_idx = 0;
     int os_idx = 0;
@@ -71,6 +76,7 @@
 // Iterate over batch elements
 L1:
     for (int i = 0; i < BATCH; i++) {
+#pragma HLS PIPELINE II = 1
     // Iterate over output classes
     L2:
         for (int j = 0; j < CLASSES; j++) {
